js/src/launchAsync.ts

```diff
@@ -151,7 +151,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             });
         }
 
-        const domain = options.customDomain ? options.customDomain : 'https://learningtools.onenote.com/learningtoolsapp/cognitive/';
+        const domain = options.customDomain ? options.customDomain : `https://${subdomain}.cognitiveservices.azure.com/immersivereader/webapp/v1.0/`;
```

Comment:
since we now use the subdomain here, let's modify the isValidSubdomain to reject !subdomain (and modify the comments above it).
we should also find a place to mention that the issueToken is deprecated with this change